lib/multi.c

```diff
@@ -1110,6 +1110,16 @@ CURLMcode curl_multi_fdset(struct Curl_multi *multi,
   return CURLM_OK;
 }
 
+#ifdef USE_WINSOCK
+static bool check_socktype_stream(SOCKET s)
```

Comment:
since this returns TRUE for TCP, maybe the function name could be named something llike `is_tcp_socket()` to make it more obvious below?

Reply:
nevermind